ui/imports/shared/controls/DisabledTooltipButton.qml

```diff
@@ -0,0 +1,61 @@
+import QtQuick 2.14
+
+import StatusQ.Controls 0.1
+
+Item {
+    id: root
+    property string aliasedObjectName
+    property string text
+    property string icon
+    property alias tooltipText: tooltip.text
+    property int buttonType: FooterTooltipButton.Normal
```

Comment:
This looks wrong, you just removed `FooterTooltipButton`

Reply:
All done @caybro